airflow-core/src/airflow/ui/public/i18n/locales/he/admin.json

```
@@ -21,11 +21,14 @@
     },
     "connection_one": "חיבור",
     "connection_other": "חיבורים",
+    "connection_two": "חיבורים",
     "delete": {
       "deleteConnection_one": "מחק חיבור אחד",
       "deleteConnection_other": "מחק {{count}} חיבורים",
+      "deleteConnection_two": "מחק שני חיבורים",
```

Not a real suggestion:
```suggestion
      "deleteConnection_two": "מחק חיבוריים",
```
🤣

---

lol nice one :)